sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/manager/DownloadManagerRequestTask.java

```diff
@@ -35,7 +37,7 @@ protected Void doInBackground(Void... params) {
         AppCenterLog.debug(LOG_TAG, "Start downloading new release from " + downloadUrl);
         DownloadManager downloadManager = mDownloader.getDownloadManager();
         DownloadManager.Request request = createRequest(downloadUrl);
-        request.setTitle(String.format("Downloading version %s (%s)", releaseDetails.getShortVersion(), releaseDetails.getVersion()));
+        request.setTitle(String.format("%s %s (%s)", mTitle, releaseDetails.getShortVersion(), releaseDetails.getVersion()));
```

Comment:
How about using https://stackoverflow.com/questions/12627457/format-statement-in-a-string-resource-file, we might have done it in the past for progress bar too.